templates/al2023/template.json

```diff
@@ -115,6 +115,7 @@
   "provisioners": [
     {
       "type": "shell",
+      "remote_folder": "{{ user `remote_folder`}}",
```

Comment:
Does Packer leave behind all the scripts it uploads to the remote_folder? It'd be nice to clean them up if `/home/ec2-user` is going to be used as `remote_folder`

Reply:
I confirmed that  `/home/ec2-user/` is clean when connecting to the Instance using the generated AMI:
```

sh-5.2$ sudo su -
[root@ip-10-0-9-125 ~]# cd /home/ec2-user/
[root@ip-10-0-9-125 ec2-user]# ls -ltrh
total 0
```

Reply:
@cartermckinnon any chance you can approve this PR and merge with updated template.json?